dwave/system/composites/tiling.py

```diff
@@ -39,25 +39,40 @@
 
 
 class TilingComposite(dimod.Sampler, dimod.Composite, dimod.Structured):
-    """Composite to tile a small problem across a Chimera-structured sampler.
-
-    Enables parallel sampling for small problems (problems that are minor-embeddable in
-    a small part of a D-Wave solver's :term:`Chimera` graph).
-
-    Notation *CN* refers to a Chimera graph consisting of an NxN grid of unit cells, where
-    each unit cell is a bipartite graph with shores of size t. The D-Wave 2000Q QPU
-    supports a C16 Chimera graph: its 2048 qubits are logically mapped into a 16x16 matrix of
-    unit cell of 8 qubits (t=4).
-
-    A problem that can be minor-embedded in a single unit cell, for example, can therefore
-    be tiled across the unit cells of a D-Wave 2000Q as 16x16 duplicates. This enables
-    sampling 256 solutions in a single call.
+    """Composite to tile a small problem across a sampler.
+
+    Enables parallel sampling for small problems (problems that are 
+    minor-embeddable in a :term:`Chimera` graph of dimensions (``sub_m``, ``sub_n``, ``t``). 
+    The sampler used can be Chimera or Pegasus structured, since Chimera graphs
+    are subgraphs of Pegasus graphs.
+
+    Notation *CN* refers to a Chimera graph consisting of an NxN grid of unit 
+    cells, where each unit cell is a bipartite graph with shores of size t. 
+    The D-Wave 2000Q QPU supports a C16 Chimera graph: its 2048 qubits are 
+    logically mapped into a 16x16 matrix of unit cell of 8 qubits (t=4).
+
+    Notation *PN* referes to a Pegasus graph consisting of 3x(N-1)x(N-1) grid 
+    of cells, where each unit cell is a bipartite graph with shore of size t, 
+    supplemented with odd couplers (see nice_coordinate definition). The 
```

Comment:
Be helpful to link to https://docs.ocean.dwavesys.com/en/stable/docs_dnx/reference/generated/dwave_networkx.pegasus_graph.html

Reply:
Is there special syntax for rendering of links in docstrings?

Reply:
:func:`dwave_networkx.pegasus_graph`

Reply:
Ah yes, forgot that was a repository reference - glad I asked!